### ISSUE TITLE
How to use AFBidLoader

### ISSUE THREAD

**Issue by MagnusWittrup-Manatee** [adform/adform-ios-sdk#36]
Hello,
I am confused as to how to display an ad acquired from AFBidLoader.
Can you please direct me to a an example of the header bidding being used?
// Magnus

**Comment by vladasdrejeris** [comment]
Hello,

Please follow our documentation: https://github.com/adform/adform-ios-sdk/wiki/Header-Bidding

**Comment by MagnusWittrup-Manatee** [comment]
Yes I have read that, but I am a bit confused as to what `bannerView` refers to?
The `AFAdInline` has no methods relating to loading a bid request.
Sorry if this is explicitly described in the documentation and I am just not seeing it.

**Comment by vladasdrejeris** [comment]
Hello, 

bidloader is useful when some other flow exists to render an ad, other ads sdk or something - the header bidding case. If you need to render an ad using our sdk there is no point in using bidloader. I would recommend not using it unless there is no other options.
A recommended way to implement header bidding setups is Prebid Mobile suite. https://docs.prebid.org/prebid-mobile/prebid-mobile.html

**Comment by MagnusWittrup-Manatee** [comment]
Thank you! 
I am still a little confused on how to actually implement this. 
I am using the Smart Display Kit SDK as my primary way of showing ads, but I would also like to be able to show AdForm ads, based on the response from the hader bidding of this SDK.
Is this at all possible or do I need to integrate Prebid Mobile to use AdForm header bidding?

//magnus

**Comment by vladasdrejeris** [comment]
Hello,

I'm not sure if I can suggest something better than https://github.com/adform/adform-ios-sdk/wiki/header-bidding#5-dfp-integration, but it seems you should use AFBidResponse.adUnitScriptRaw string and render this using your primary ads SDK instead of appending it as targeting for DFP.
For additional information on how to render it with Smart SDK, you should contact them.
